cmd/lifecycle/cli/command.go

```diff
@@ -57,6 +57,12 @@ func Run(c Command, withPhaseName string, asSubcommand bool) {
 	}
 	cmd.DefaultLogger.Debugf("Starting %s...", withPhaseName)
 
+	for _, arg := range flagSet.Args() {
+		if arg[0:1] == "-" {
+			cmd.DefaultLogger.Warnf("Warning: unconsumed flag-like positional arg: \n\t%s\n\t This will not be interpreted as a flag.\n\t Did you mean to put this before the first positional argument?", arg)
```

Comment:
Oh, this was already merged, but I think we could remove the  `Warning:` because we are using `Warnf`

Reply:
ah yeah... @jjbustamante  you're right - it's a "Warning: Warning: "  😅   maybe we say it extra to make up for all the years we were silent about it???